docs/pages/components/table.md

```diff
@@ -19,11 +19,11 @@ A table is a set of tabular data. Line items can support data, images and action
 <table class="fd-table">
     <thead class="fd-table__header">
         <tr class="fd-table__row">
-            <th class="fd-table__cell" scope="col"></th>
-            <th class="fd-table__cell" scope="col">Column Header</th>
-            <th class="fd-table__cell" scope="col">Column Header</th>
-            <th class="fd-table__cell" scope="col">Column Header</th>
-            <th class="fd-table__cell" scope="col">Column Header</th>
+            <th class="fd-table__cell fd-table__cell--header" scope="col"></th>
```

Comment:
Why do we need this class?

Reply:
`th` has different padding than `td` and since we are using the class `fd-table_cell` for td, i created a modifier `--header` that applies this padding for `th`

Reply:
@saad-mo instead of creating an extra class `fd-table_cell--header`, can you just add the `padding-top` and `padding-bottom` in the location of under cell styles in the header section? (between lines 78-88 in table.scss).